crates/ruff/src/importer/insertion.rs

```diff
@@ -67,12 +67,10 @@ impl<'a> Insertion<'a> {
             TextSize::default()
         };
 
-        // Skip over any comments and empty lines.
-        for (tok, range) in
-            lexer::lex_starts_at(locator.after(location), Mode::Module, location).flatten()
-        {
-            if matches!(tok, Tok::Comment(..) | Tok::Newline) {
-                location = locator.full_line_end(range.end());
+        // Skip over commented lines.
+        for line in locator.after(location).universal_newlines() {
+            if line.trim_start().starts_with('#') {
+                location = locator.full_line_end(location);
```

Comment:
Line exposes the `start` and end offsets to avoid re-scanning the text to find the end of the line

```suggestion
                location = line.full_end();
```

Is it okay that `location` now is a relative offset? If not, use UniversalNewlinesIter::with_offset(locator.after(location), location)`

Reply:
Thanks, fixed.